apps/dg/controllers/document_controller.js

```diff
@@ -206,6 +206,9 @@ DG.DocumentController = SC.Object.extend(
           return;
         }
         var name = this.get('documentName');
+        if (name === null) {
```

Comment:
A better test would be SC.empty which checks for null, undefined, and empty string.

Reply:
Got it.  Changes added to use SC.empty.